packages/cli/src/utils/files.js

```diff
@@ -207,9 +207,17 @@ function* walkDir(dir) {
   const entries = fs.readdirSync(dir, { withFileTypes: true });
   for (const entry of entries) {
     if (entry.isDirectory()) {
-      const subDir = path.join(entry.parentPath, entry.name);
+      const subDir = path.join(
+        entry.parentPath || entry.path || dir,
```

Comment:
Why not use only the `dir` that is always defined?

Reply:
Good call. I wanted to be careful not to change existing code paths by accident. But you're right, we should remove redundant code paths. Fixed it [3bf5c04](https://github.com/zapier/zapier-platform/pull/1087/commits/3bf5c04af3f5bb1999f0e83003536fd0713342ec).